src/containers/Home.tsx

```diff
@@ -101,7 +121,7 @@ export const Home = (): JSX.Element => {
       setLongitude(0);
       setLatitude(0);
       setPermissions(false);
-      setSafeplaces([]);
+      // setSafeplaces([]);
```

Comment:
If you are not planning on uncommenting this function again in the future you can remove it.